mod-calendar/src/main/resources/bama/mod-calendar/features/calendarPeriods.feature

```diff
@@ -285,8 +281,8 @@ Feature: Calendar periods
     Given path 'calendar/periods/' + servicePointId + '/period'
     And request createPeriodRequest
     When method POST
-    Then status 422
```

Comment:
`409 Conflict` is now used for overlapping calendars with more information about the conflict itself.